data/t.json

```diff
@@ -104025,7 +104025,7 @@
             {
                 "id": "b4e163ba3e",
                 "def": "the feeling of being so happy, or laughing so much that you cry",
-                "example": "Seeing that hilarious video made me so 😂.",
+                "-example": "Seeing that hilarious video made me so 😂.",
```

Comment:
Hmm... this one is interesting as we do also have emoji officially supported by the dictionary!

Reply:
I guess LateX doesn't support emoji? :P